curryer/compute/spatial.py

```diff
@@ -76,13 +130,49 @@ def pixel_vectors(instrument: Union[int, str, spicierpy.obj.Body]) -> tuple[int,
         vectors = vectors.reshape((count, 3))
 
     except NotFoundError:
+        logger.debug("Pixel vector data not found. Using the instrument boresight.")
         count = 1
         vectors = np.array([boresight_vector])
 
     logger.debug("Read [%d] pixel vectors for instrument [%s]", count, instrument)
     return count, vectors
 
 
+def pixel_vectors(instrument: Union[int, str, spicierpy.obj.Body]) -> tuple[int, np.ndarray]:
+    """Load the pixel or boresight vector(s) for a given instrument.
+
+    Boresight vector is queried from the instrument kernel, but superseded by
+    pixel vectors if they are available.
+
+    Pixel vectors are queried from the SPICE kernel variable pool using the
+    non-standard (but established) variables:
+        INS{instrument_id}_PIXEL_COUNT
+        INS{instrument_id}_PIXEL_VECTORS
+    where the former is the pixel index and the latter is three doubles.
+
+    Parameters
+    ----------
+    instrument : spicierpy.obj.Body or int or str
+        Instrument to query details about.
+
+    Returns
+    -------
+    int
+        Number of boresight (1) or pixel vectors (1+).
+    np.ndarray
+        2D array of boresight (1x3) or pixel vectors (1+,3).
+
+    """
+    # Print deprecation warning and call the new function.
+
+    logger.warning(
+        "`pixel_vectors` is deprecated and will be removed in a future release. "
+        "Please use `get_instrument_kernel_pointing_vectors` instead.",
+        stacklevel=2,
+    )
```

Comment:
I'd suggest issuing a DeprecationWarning here with instructions for how to update user code to the new function.
https://docs.python.org/3/library/exceptions.html#DeprecationWarning